reference/ToDo/src/ToDo/Assets/Icons/icon_foreground.svg

```diff
@@ -1,137 +1,9 @@
wONGR0YiNAADAI4IjQAOiI6MQGwEAgGcER4AmREeuJjYCAABrCI4AjYiOXEVsBAAA1hIcAZoRHTmb2AgAAGwhOAI0JDpyFrERAADYSnAEaEp05GhiIwAAsIfgCNCY6MhRxEYAAGAvwRGgOdGRamIjAADwCsERIIDoSBWxEQAAeJXgCBBCdORVYiMAAFBBcAQIIjqyl9gIAABUERwBwoiObCU2AgAAlQRHgECiI2uJjQAAQDXBESCU6MgzYiMAAHAEwREgmOjIPWIjAABwFMERIJzoyEdiIwAAcCTBEWACoiO/ITYCAABHExwBJiE6IjYCAABnEBwBJiI6zktsBAAAziI4AkxGdJyP2AgAAJxJcASYkOg4D7ERAAA4m+AIMCnRMZ/YCAAAXEFwBJiY6JhLbAQAAK4iOAJMTnTMIzYCAABXEhwBEB2DiI0AAMDVBEcAPhEd+xMbAQCAEQiOAHwhOvYlNgIAAKMQHAF4R3TsR2wEAABGIjgC8BXRsQ+xEQAAGI3gCMBNouP4xEYAAGBEgiMAd4mO4xIbAQCAUQmOADwkOo5HbAQAAEYmOALwlOg4DrERAAAYneAIwCqi4/XERgAAoAPBEYDVRMfriI0AAEAXgiMAm4iO5xMbAQCATgRHADYTHc8jNgIAAN0IjgDsIjoeT2wEAAA6EhwB2E10PI7YCAAAdCU4AvAS0bGe2AgAAHQmOALwMtGxjtgIAAB0JzgCUEJ0fJ3YCAAAJBAcASgjOu4nNgIAACkERwBKiY7biY0AAEASwRGAcqLjemIjAACQRnAE4BCi43NiIwAAkEhwBOAwouN9YiMAAJBKcATgUKLj18RGAAAgmeAIwOFExx+JjQAAQDrBEYBTiI5iIwAAMAfBEYDTzBwdxUYAAGAWgiMAp5oxOoqNAADATARHAE43U3QUGwEAgNkIjgBcYoboKDYCAAAzEhwBuExydBQbAQCAWQmOAFwqMTqKjQAAwMwERwAulxQdxUYAAGB2giMAQ0iIjmIjAACA4AjAQDpHR7ERAADgB4IjAEPpGB3FRgAAgB8JjgAMp1N0FBsBAADeExwBGFKH6Cg2AgAAfE1wBGBYI0dHsREAAOA2wRGAoY0YHcVGAACA+wRHAIY3UnQUGwEAAB4THAFoYYToKDYCAAA8JzgC0MaV0VFsBAAAWEdwBKCVK6Kj2AgAALCe4AhAO2dGqjCK7gAABSZJREFUR7ERAABgG8ERgJbOiI5iIwAAwHaCIwBtHRkdxUYAAIB9BEcAWjsiOoqNAAAA+wmOALRXGR3FRgAAgNcIjgBEqIiOYiMAAMDrBEcAYrwSHcVGAACAGoIjAFH2REexEQAAoI7gCECcLdFRbAQAAKglOAIQaU10FBsBAADqCY4AxHoUHcVGAACAYwiOAES7FR3FRgAAgOMIjgDEexsdxUYAAIBj/dZ3333nxABM4e9//q//+HvLf/2t2AgAAHAcwREAAAAAKOOlGgAAAAAoIzgCAAAAAGUERwAAAACgjOAIAAAAAJQRHAEAAACAMoIjAAAAAFBGcAQAAAAAygiOAAAAAEAZwREAAAAAKCM4AgAAAABlBEcAAAAAoIzgCAAAAACUERwBAAAAgDKCIwAAAABQRnAEAAAAAMoIjgAAAABAGcERAAAAACgjOAIAAAAAZQRHAAAAAKCM4AgAAAAAlBEcAQAAAIAygiMAAAAAUEZwBAAAAADKCI4AAAAAQBnBEQAAAAAoIzgCAAAAAGUERwAAAACgjOAIAAAAAJQRHAEAAACAMoIjAAAAAFBGcAQAAAAAygiOAAAAAEAZwREAAAAAKCM4AgAAAABlBEcAAAAAoIzgCAAAAACUERwBAAAAgDKCIwAAAABQRnAEAAAAAMoIjgAAAABAGcERAAAAACgjOAIAAAAAZQRHAAAAAKCM4AgAAAAAlBEcAQAAAIAygiMAAAAAUEZwBAAAAADKCI4AAAAAQBnBEQAAAAAoIzgCAAAAAGUERwAAAACgjOAIAAAAAJQRHAEAAACAMoIjAAAAAFBGcAQAAAAAygiOAAAAAEAZwREAAAAAKCM4AgAAAABlBEcAAAAAoIzgCAAAAACUERwBAAAAgDKCIwAAAABQRnAEAAAAAMoIjgAAAABAGcERAAAAACgjOAIAAAAAZQRHAAAAAKCM4AgAAAAAlBEcAQAAAIAygiMAAAAAUEZwBAAAAADKCI4AAAAAQBnBEQAAAAAoIzgCAAAAAGUERwAAAACgjOAIAAAAAJQRHAEAAACAMoIjAAAAAFBGcAQAAAAAygiOAAAAAEAZwREAAAAAKCM4AgAAAABlBEcAAAAAoIzgCAAAAACUERwBAAAAgDKCIwAAAABQRnAEAAAAAMoIjgAAAABAGcERAAAAACgjOAIAAAAAZQRHAAAAAKCM4AgAAAAAlBEcAQAAAIAygiMAAAAAUEZwBAAAAADKCI4AAAAAQBnBEQAAAAAoIzgCAAAAAGUERwAAAACgjOAIAAAAAJQRHAEAAACAMoIjAAAAAFBGcAQAAAAAygiOAAAAAEAZwREAAAAAKCM4AgAAAABlBEcAAAAAoIzgCAAAAACUERwBAAAAgDKCIwAAAABQRnAEAAAAAMoIjgAAAABAGcERAAAAACgjOAIAAAAAZQRHAAAAAKCM4AgAAAAAlBEcAQAAAIAygiMAAAAAUEZwBAAAAADKCI4AAAAAQBnBEQAAAAAoIzgCAAAAAGUERwAAAACgjOAIAAAAAJQRHAEAAACAMoIjAAAAAFBGcAQAAAAAygiOAAAAAEAZwREAAAAAKCM4AgAAAABlBEcAAAAAoIzgCAAAAACUERwBAAAAgDKCIwAAAABQRnAEAAAAAMoIjgAAAABAGcERAAAAACgjOAIAAAAAZQRHAAAAAKCM4AgAAAAAlBEcAQAAAIAygiMAAAAAUEZwBAAAAADKCI4AAAAAQBnBEQAAAAAoIzgCAAAAADWWZfl/U3AQ6co2k8UAAAAASUVORK5CYII="/>
```

Comment:
This just looks like it's just embedding an image inside the svg? Not sure why it's so different from the old svg contents

Reply:
Hum good catch @kazo0, I will double check as I exported these svg from Figma directly

Reply:
I am only left with updating those SVG. I will quickly sync with @NVLudwig tomorrow to see why when I export with Figma I have these data for the SVG files.

Reply:
@kazo0 SVG files are updated now.